packages/pulpcore/python-aiohttp/python-aiohttp.spec

```diff
@@ -9,6 +9,7 @@ Summary:        Async http client/server framework (asyncio)
 License:        Apache 2
 URL:            https://github.com/aio-libs/aiohttp
 Source0:        https://files.pythonhosted.org/packages/source/a/%{pypi_name}/%{pypi_name}-%{version}.tar.gz
+Patch0:         allow-larger-headers.patch
```

Comment:
is something like this required as well in %prep? 

%patch0 -p1

i always use it when adding a patch

Reply:
We use `%autosetup` so maybe that already takes care of it?

Reply:
I left it as is because I took it off from here https://github.com/theforeman/foreman-packaging/blob/rpm/develop/packages/pulpcore/python-requests/python-requests.spec#L12

Reply:
They have a different autosetup line with `-p1`: https://github.com/theforeman/foreman-packaging/blob/3964a5c2d1aafb2e2d3092ab428de7ce80306bc4/packages/pulpcore/python-requests/python-requests.spec#L38